components/Main.jsx

```diff
@@ -1,84 +1,103 @@
   return (
-    <Box p={[2, 3, 5]} mb={[0, 4]}>
+    <Box
+      display='flex'
+      flexWrap='wrap'
+      alignItems='center'
+      justifyContent='center'
+      flexGrow='1'
+      p={[2, 3, 5]}
+    >
       <Box
         display='flex'
+        width={['100%', '100%', '45%', '40%']}
         flexDirection='column'
         alignItems='flex-start'
         alignContent='center'
+        mx={3}
         mb={4}
       >
-        <Menu
-          display='flex'
-          justifyContent='space-between'
-          width='100%'
-          alignItems='center'
-        >
-          <MenuItem>
-            <Box
-              display='flex'
-              justifyContent='center'
-              alignItems='center'
-              borderRadius={3}
-              bg='core.primary'
-              borderRadius={3}
-              height={7}
-            >
-              <IconGlif width='48px' height='48px' fill='white' />
-            </Box>
-          </MenuItem>
-          {/* <MenuItem>
-            <NodeConnectedGlyph apiAddress={process.env.LOTUS_NODE_JSONRPC} />
-          </MenuItem> */}
-        </Menu>
+        <HeaderGlyph
+          alt='Source: https://unsplash.com/photos/g2Zf3hJyYAc'
+          text='Faucet'
+          imageUrl='/imgfaucet.jpg'
+        />
+
         <Box
           display='flex'
           flexDirection='column'
-          mt={4}
+          mt={[2, 4, 4]}
           alignSelf='center'
-          alignItems='center'
-          textAlign='center'
+          textAlign='left'
         >
-          <Title fontSize={5}>Filecoin faucet</Title>
-          <Text maxWidth={15}>
-            This app provides small amounts of testnet FIL to anyone who has a
-            GitHub account to make testing and experimentation easy.
-            <br />
-            <br />
-            Non-miners can only use this faucet once.
-            <br />
-            <br />
-            Miners with more power will receive more testnet FIL from this
-            faucet.
-          </Text>
+          <Header>Quickly, easily receive testnet Filecoin</Header>
+          <Title mt={3} lineHeight='140%'>
+            Receive small amounts of testnet FIL to help your testing and
+            experimentation.
+          </Title>
+          <Box mt={[2, 3, 4, 6]} maxWidth={12}>
+            <Label color='core.darkgray' textAlign='left' mt={3}>
+              <Highlight>Miners</Highlight>
+              You can reuse this faucet by entering in your miner ID
+            </Label>
+            <Label color='core.darkgray' textAlign='left' mt={3}>
+              <Highlight>Not a miner?</Highlight>
+              t1 and t3 addresses can only use this faucet once
+            </Label>
+          </Box>
         </Box>
+        <Title
+          display={['inline-block', 'inline-block', 'none']}
+          width='100%'
+          textAlign='center'
+          fontSize={6}
+          css={`
+            font-family: 'system-ui';
+          `}
```

Comment:
Smelly

Reply:
lol I know. I thought it was reasonable given how tiny it is.

Reply:
Got a better idea..

Reply:
Removed